_source/user-guide/integrations/endpoints.md

```diff
@@ -39,3 +46,9 @@ Alerts are sent on ports 80 and 443 only.
 Setting a custom port as part of an endpoint URL
 will result in alerts not being sent to said endpoint.
 {:.info-box.important}
+
+* To test an endpoint after it is created, open it in edit mode. 
+Click **Run the test** and check the recepient for the test message. 
+Logz.io will notify you if the message couldn't be sent, 
+but you'll need to confirm that the message was received. 
```

Comment:
![image](https://user-images.githubusercontent.com/6845404/76897898-4b421e80-689d-11ea-93ff-f73345180258.png)

Move these above the note on line 46, so that the note doesn't break your bullet list.